src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -1,29 +1,28 @@
 package com.itacademy.eshop.services;
 
 import com.itacademy.eshop.product.Product;
+import com.itacademy.eshop.product.types.Category;
 import com.itacademy.eshop.shop.Eshop;
 
+import java.sql.Array;
 import java.util.ArrayList;
 
 public class ShopService {
-    /*** This class is used for creating new e-shop object and initializing it with some products.
-     */
-
     public Eshop createShop() {
-        /** Should create new e-shop object and initialize it with some products.
-         * Should return e-shop object.
-         */
-       ArrayList<Product> products = createProducts();
-        throw new UnsupportedOperationException("createShop() is not implemented yet");
+        ArrayList<Product> products = createProducts();
+
+        return new Eshop("Test", products);
     }
 
     private ArrayList<Product> createProducts() {
-        /**
-         * Should return a list of products. Should contain at least 6 products,
-         * one per each category in Category enum.
-         * Product should not have any reviews.
-         */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> arrayList = new ArrayList<>();
```

Comment:
Avoid using type names as parameter names. opt out to use the name of the objects it hold. In this case `products`